dashboard.html

```
@@ -0,0 +1,867 @@
+    }
+
+    function trackAction(company) {
+        const command = `/career-ops tracker --details "${company}"`;
+        logToTerminal('tracker', `Fetching detailed status for ${company}...`);
+        copyToClipboard(command);
+    }
+</script>
+
+</body>
+</html>
+tml>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)